File: Beginner/cV/cV.c
#include <stdio.h>
#include <assert.h>

int main(void){
	int T,N,count;

	scanf("%d",&T);
	assert(T>=1 && T<=100);

	while(T>0){
		scanf("%d",&N);
		assert(N>=1 && N<=100);

		char S[N];
		scanf("%s",S);

		for(int i=0;i<N;i++){
			assert(S[i]>=97 && S[i]<=122);
		}

		count = 0;

		if(N==1){
			printf("%d\n",count);
		}else{
			for(int i=0;i<N-1;i++){
				if(S[i]!='a' || S[i]!='e' || S[i]!='i' || S[i]!='o' || S[i]!='u'){
					if(S[i+1]=='a' || S[i+1]=='e' || S[i+1]=='i' || S[i+1]=='o' || S[i+1]=='u'){
						count++;
					}
				}
			}

			printf("%d\n",count);
		}

		T--;
	}

	return 0;
}

File: Beginner/cV/cV.java
import java.util.Scanner;

public class cV{
	public static void main(String[] args) {
		int T,N,count;
		String S;
		Scanner scan = new Scanner(System.in);

		T = scan.nextInt();
		assert(T>=1 && T<=100) : "T must be 1 <= T <= 100";

		while(T>0){
			N = scan.nextInt();
			assert(N>=1 && N<=100) : "N must be 1 <= N <= 100";

			S = scan.next();

			for(int i=0;i<S.length();i++){
				assert(S.charAt(i)>=97 && S.charAt(i)<=122) : "Each character must be lower case English letters";
			}

			count = 0;

			if(N==1){
				System.out.println(count);
			}else{
				for(int i=0;i<S.length()-1;i++){
					if(S.charAt(i)!='a' || S.charAt(i)!='e' || S.charAt(i)!='i' || S.charAt(i)!='o' || S.charAt(i)!='u'){
						if(S.charAt(i+1)=='a' || S.charAt(i+1)=='e' || S.charAt(i+1)=='i' || S.charAt(i+1)=='o' || S.charAt(i+1)=='u'){
							count++;
						}
					}
				}

				System.out.println(count);
			}

			T--;
		}
	}
}